src/bioregistry/export/prefix_maps.py

```diff
@@ -15,25 +15,56 @@
     CONTEXT_OBO_PATH,
     CONTEXT_OBO_SYNONYMS_PATH,
     EXPORT_CONTEXTS,
+    SHACL_TURTLE_PATH,
 )
 from bioregistry.schema import Collection
 
 
 @click.command()
-def generate_context_json_ld():
+def generate_contexts():
     """Generate various JSON-LD context files."""
-    _write_prefix_map(CONTEXT_BIOREGISTRY_PATH, get_prefix_map())
+    prefix_map = get_prefix_map()
+    _write_prefix_map(CONTEXT_BIOREGISTRY_PATH, prefix_map)
+    _write_shacl(..., prefix_map)
+
     _write_prefix_map(CONTEXT_OBO_PATH, get_obofoundry_prefix_map())
     _write_prefix_map(CONTEXT_OBO_SYNONYMS_PATH, get_obofoundry_prefix_map(include_synonyms=True))
 
     for key, collection in bioregistry.read_collections().items():
         name = collection.context
         if name is None:
             continue
-        with EXPORT_CONTEXTS.joinpath(name).with_suffix(".context.jsonld").open("w") as file:
+        context_path_stub = EXPORT_CONTEXTS.joinpath(name)
+        with context_path_stub.with_suffix(".context.jsonld").open("w") as file:
             json.dump(fp=file, indent=4, sort_keys=True, obj=get_collection_jsonld(key))
 
 
+@click.command()
+def generate_shacl_prefixes():
+    """Generate a SHACL prefixes file."""
+    # TODO: store in context folder
+    # TODO put this inside :func:`generate_contexts`
+    # TODO generate several variants corresponding to each prefix map in the above function
+    _write_shacl(SHACL_TURTLE_PATH, get_prefix_map())
+
+
+def _write_shacl(path: Path, prefix_map: Mapping[str, str]) -> None:
+    text = """@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+[
+    sh:declare
+{entries}
+] ."""
+    # Todo: Should we really uppercase prefixes?
+    path.parent.mkdir(parents=True, exist_ok=True)
+    entries = ",\n".join(
+        f'    [ sh:prefix "{prefix.upper()}" ; sh:namespace "{url}" ]'
+        for prefix, url in prefix_map.items()
```

Comment:
rename `url` to `uri_format`

Reply:
Fixed in https://github.com/biopragmatics/bioregistry/pull/318/commits/ef82d08b43d2cbea957a7e09e366e12fc9fc6759